docs/developer/architecture/logs.rst

```diff
@@ -197,6 +197,444 @@ overview page.
 A Grafana dashboard gathering health/performance KPIs, as well as alerts for
 log centralization service is available when deploying/upgrading MetalK8s.
 
+Components
+----------
+
+Our Log Centralization system can be splitted into several components as
+follows:
+
+.. parsed-literal::
+
+  +-----------+     +-------------+     +----------+     +---------+
+  | Collector | ==> | Distributor | ==> | Ingester | <=> | Querier |
+  +-----------+     +-------------+     +----------+     +---------+
+
+.. todo:: do a better schema. :)
```

Comment:
Yes, that's my point, because here we don't really understand interactions when there is multiple Loki instances, for example, or do each collector write to one or multiple distributors ?